nixos/modules/services/audio/mpd.nix

```diff
@@ -123,10 +133,24 @@ in {
 
   config = mkIf cfg.enable {
 
+    systemd.sockets.mpd = mkIf cfg.startWhenNeeded {
+      description = "Music Player Daemon Socket";
+      wantedBy = [ "sockets.target" ];
+      listenStreams = [
+        "/run/mpd/socket"
+        "${cfg.network.listenAddress}:${toString cfg.network.port}"
+      ];
```

Comment:
This is more of a general problem, but hardcoding "/run/mpd/socket" here and in other places is a not a modular solution. 

It means that if I later want to add another service which is also called `mpd`, but does something completely different that I need to care about the existence of this particular service; something a well designed system would not require.